opm/autodiff/RedistributeDataHandles.hpp

```diff
@@ -219,7 +219,7 @@ class BlackoilStateDataHandle
     {
         if ( T::codimension == 0)
         {
-            return 2 * sendState_.numPhases() +4+2*sendGrid_.numCellFaces(e.index());
+            return 2 * sendState_.numPhases() + 5 + 2*sendGrid_.numCellFaces(e.index());
```

Comment:
I do not see a way around counting the data that we send per cell unless we change the way data is represented (e.g. one struct will all data per cell and then one vector of it). But then we might produce a lot of cache missing in the compute intensive code that of only uses one value per cell at a time.